functests/dpdk/dpdk.go

```diff
@@ -51,40 +51,38 @@ var _ = Describe("dpdk", func() {
 // checkRxTx parses the output from the DPDK test application
 // and verifies that packets have passed the NIC TX and RX queues
 func checkRxTx(out string) {
-	str := strings.Split(out, "\n")
-	for i := 0; i < len(str); i++ {
-		if strings.Contains(str[i], logEntry) {
-			i++
-			d := getNumberOfPackets(str, i)
-			Expect(d).Should(BeNumerically(">", 0), "number of received packets should be greater than 0")
-
-			i++
-			d = getNumberOfPackets(str, i)
-			Expect(d).Should(BeNumerically(">", 0), "number of transferred packets should be greater than 0")
+	lines := strings.Split(out, "\n")
+	for i, line := range lines {
```

Comment:
Assert here that len(lines) >= 3

Reply:
@fedepaol, why we need to add this assertion. We already have two verifications:
1. We check whether log contains an appropriate header (and if it does, we are good to go)
2. We check the first field for an appropriate pattern

Reply:
Well, my reasoning was that if the file is malformed, the test would panic in `d := getNumberOfPackets(lines[i+1], "RX")` as the `i+1` assumes the next line to be there.